average_height.py

```diff
@@ -0,0 +1,21 @@
+# 🚨 Don't change the code below 👇
+student_heights = input("Input a list of student heights ").split()
+for n in range(0, len(student_heights)):
+  student_heights[n] = int(student_heights[n])
+# 🚨 Don't change the code above 👆
+###Objective
+#You are going to write a program that calculates the average student height from a List of heights.
+#mportant You should not use the sum() or len() functions in your answer. You should try to replicate their functionality using what you have learnt about for loops.
+#Write your code below this row 👇
+sum_of_heights = 0
+index = 0
```

Comment:
Index is a bit cryptic. It's this a count of the number of heights seen? I think `count` would be a better name.

Reply:
Thanks, good point! :)